### PR TITLE
Release/1.3.4

### PR DESCRIPTION
**Pull request by captn3m0**

### CODE REVIEW

File: src/banknames.json [hunk 1]
```diff
@@ -805,6 +808,7 @@
   "NILX": "Nilkanth Co-operative Bank",
   "NIRX": "Nirmal Urban Co-operative Bank Nagpur",
   "NIUX": "Nileshwar Co-operative Urban Bank",
+  "NJBK": "Nav Jeevan Co-operative Bank",
   "NJCX": "Nav Jeevan Co-operative Bank",
```
Comment (patlola) [inline src/banknames.json]: @captn3m0 need to do a cleanup of the banknames.json once. some sublets like the above are now proper banks we have to remove all the sublet bank codes

Comment (captn3m0) [reply src/banknames.json]: I don't want to remove entries from `banknames.json` unless it was added in an error. Learned this the hard way after I'd removed

>"VYSA": "ING Vysya Bank",

long back. But yeah, the sublet confusion is getting bigger, and the exceptions list is getting confusing. Need a better solution.

**Comment by captn3m0** [comment]
@patlola Due to some parsing error somewhere, this currently marks `IFSCCODE` as a valid IFSC. Don't have time to debug that, do you think we can do a release with this?

I'll ensure it doesn't reach the API.

**Comment by patlola** [comment]
yeah, sure. we can push this as far as the generated files are good to go. will check once on why this parsing error is happening.

**Pull request merged**